From: Steve Langasek <steve.langasek@canonical.com>
Date: Sat, 21 Jan 2012 17:23:25 -0500
Subject: Multiarch support in util-linux build

Addresses-Debian-Bug: 631175
Signed-off-by: LaMont Jones <lamont@debian.org>

Conflicts:
	debian/control
	debian/libblkid-dev.files
	debian/libblkid1.files
	debian/libmount-dev.files
	debian/libmount1.files
	debian/libuuid1.files
	debian/rules
	debian/uuid-dev.files
---
 Makefile.am          | 1 +
 configure.ac         | 4 ++--
 libblkid/blkid.pc.in | 1 +
 libmount/mount.pc.in | 1 +
 libuuid/uuid.pc.in   | 1 +
 5 files changed, 6 insertions(+), 2 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index 391de3d..2ac4d5e 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -117,6 +117,7 @@ edit_cmd = sed \
 	 -e 's|@usrlib_execdir[@]|$(usrlib_execdir)|g' \
 	 -e 's|@usrbin_execdir[@]|$(usrbin_execdir)|g' \
 	 -e 's|@usrsbin_execdir[@]|$(usrsbin_execdir)|g' \
+	 -e 's|@libexecdir[@]|$(libexecdir)|g' \
 	 -e 's|@VERSION[@]|$(VERSION)|g' \
 	 -e 's|@ADJTIME_PATH[@]|$(ADJTIME_PATH)|g' \
 	 -e 's|@LIBUUID_VERSION[@]|$(LIBUUID_VERSION)|g' \
diff --git a/configure.ac b/configure.ac
index 6bc9efd..ff6fae5 100644
--- a/configure.ac
+++ b/configure.ac
@@ -78,7 +78,6 @@ AS_CASE([$prefix:$localstatedir],
 )
 AC_SUBST([localstatedir])
 
-
 # The original default values of {bin,sbin,lib}dir
 usrbin_execdir='${exec_prefix}/bin'
 AC_SUBST([usrbin_execdir])
@@ -89,9 +88,10 @@ AC_SUBST([usrsbin_execdir])
 AS_CASE([$libdir],
  ['${exec_prefix}/'* | '${prefix}/'* | /usr/* | "${prefix}"/* ],
    [usrlib_execdir=$libdir],
- [usrlib_execdir='${exec_prefix}'$libdir]
+ [usrlib_execdir='${libexecdir}']
 )
 AC_SUBST([usrlib_execdir])
+AC_SUBST([libexecdir])
 
 
 AM_PROG_CC_C_O
diff --git a/libblkid/blkid.pc.in b/libblkid/blkid.pc.in
index 40ec8a9..b89b136 100644
--- a/libblkid/blkid.pc.in
+++ b/libblkid/blkid.pc.in
@@ -1,5 +1,6 @@
 prefix=@prefix@
 exec_prefix=@exec_prefix@
+libexecdir=@libexecdir@
 libdir=@usrlib_execdir@
 includedir=@includedir@
 
diff --git a/libmount/mount.pc.in b/libmount/mount.pc.in
index 2c32797..b775b14 100644
--- a/libmount/mount.pc.in
+++ b/libmount/mount.pc.in
@@ -1,5 +1,6 @@
 prefix=@prefix@
 exec_prefix=@exec_prefix@
+libexecdir=@libexecdir@
 libdir=@usrlib_execdir@
 includedir=@includedir@
 
diff --git a/libuuid/uuid.pc.in b/libuuid/uuid.pc.in
index 875de19..a12d432 100644
--- a/libuuid/uuid.pc.in
+++ b/libuuid/uuid.pc.in
@@ -1,5 +1,6 @@
 prefix=@prefix@
 exec_prefix=@exec_prefix@
+libexecdir=@libexecdir@
 libdir=@usrlib_execdir@
 includedir=@includedir@
 
